appveyor.yml

```diff
@@ -4,6 +4,7 @@ image: Visual Studio 2015
 
 cache:
   - '%LocalAppData%\Microsoft\dotnet'
+  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
I think we need to merge to see if caching speeds up CI at all before talking too much about cache invalidation policies.

Cache invalidation is manual but simple - process is documented here: https://www.appveyor.com/docs/build-cache/